sdk/cognitiveservices/Azure.AI.Speech.Transcription/samples/Sample_1_TranscribeLocalFile.md

```diff
@@ -0,0 +1,50 @@
+# Transcribe a Local File
+
+This sample shows how to transcribe a local file using the `Azure.AI.Speech.Transcription` SDK.
+
+## Create a Transcription Client
+
+To create a TranscriptionClient, you will need the service endpoint and credentials of your SpeechService resource. You can specify the service version by providing a TranscriptionClientOptions instance.
```

Comment:
Two words. And likely we'll want to focus on an AI Foundry resource, or at least mention it.